<launch>
        <param name="robot_description" command="$(find xacro)/xacro.py '$(find example6_urdf)/urdf/mobman_jointlink.xacro'" />
	<node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
	<node name="joint_publisher" pkg="example6_urdf" type="joint_publisher" />
</launch>
